content/en/continuous_integration/guides/ci_visibility_rum_integration.md

```diff
@@ -0,0 +1,30 @@
+---
+title: CI Visibility - RUM integration
+kind: guide
+aliases: /continuous_integration/guides/ci_visibility_rum_integration/
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+### Browser tests and RUM
+
+If your page under test is instrumented using [RUM][1], your `cypress` test results and their generated RUM browser sessions and session replays will be automatically linked. A new **Browser Sessions** tab appears in the browser test's test detail:
```

Comment:
`If your page under test` 

Not sure exactly what this means

Reply:
this is the page you're running your browser tests against. See usage in cypress docs for example: https://docs.cypress.io/faq/questions/using-cypress-faq#Can-Cypress-be-used-for-performance-testing